src/pages/derivx/_selling-points.tsx

```diff
@@ -48,30 +49,29 @@ const StyledSectionContainer = styled(SectionContainer)`
 `
 
 const selling_points: SellingPointsType[] = [
-    {
-        title: '100+',
-        subtitle: <Localize translate_text="tradable assets" />,
-    },
-    {
-        title: '24/7',
-        subtitle: <Localize translate_text="trading" />,
-    },
-    {
-        title: <Localize translate_text="Zero" />,
-        subtitle: <Localize translate_text="commission" />,
-    },
+    { title: '100+', subtitle: '_t_tradable assets_t_' },
+    { title: '24/7', subtitle: '_t_trading_t_' },
+    { title: '_t_Zero_t_', subtitle: '_t_commission_t_' },
 ]
 
 const SellingPoints = () => {
     return (
         <StyledSectionContainer padding="40px 0" background="grey-25">
             <Container>
                 <Flex tablet_direction="column">
-                    {selling_points.map((selling_point, index) => {
+                    {selling_points.map(({ title, subtitle }, index) => {
                         return (
                             <StyledFlex tablet_direction="column" key={index}>
```

Comment:
Can we use `title` as `key`?

Reply:
Yes @mohammad-hashemi-deriv will change that